client/src/components/profileInfoEdit/ProfileResetModal.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Delete Alert Modal component display a modal and ask for confirmation when the delete button is clicked
+ */
+
+import React from 'react';
+import { Button, Modal } from 'react-bootstrap';
+import { useSelector, useDispatch } from 'react-redux';
+import { PropTypes } from 'prop-types';
+import { logout } from '../../actions/auth';
+import { resetTeamMember, resetTeamData, deleteGHPages } from '../../actions/team';
+import { resetHomepage } from '../../actions/homepage';
+import { resetWebPage } from '../../actions/website';
+
+const ProfileResetModal = ({ resetAlert, setResetAlert, type }) => {
+  const { teamId } = useSelector((state) => state.team);
+  const dispatch = useDispatch();
+
+  const isDeleteFlag = type;
+  const titleMessage = isDeleteFlag ? 'Delete Account' : 'Reset Account Data';
+  const warningMessage = isDeleteFlag ? 'Are you sure you want to delete your account? ' : 'Are you sure you want to reset your account? ';
+  const buttonName = isDeleteFlag ? 'Delete' : 'Reset';
+  const warningContent = isDeleteFlag ? 'All significant data will be reset!' : 'All significant data will be deleted including your account data!';
```

Comment:
I think the text should be swapped: 
![image](https://user-images.githubusercontent.com/40584352/135184212-ae90d628-7d1f-403a-aafc-bd66fd5dadd4.png)
When you're deleting the account, the second line should show rather than the first?